### PR TITLE
More comfortable variable num dice

### PR DESCRIPTION
**Pull request by rocketkai**

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -1,4 +1,307 @@
...
+// The game starts with Player 1.
+var currPlayer = 1;
+
+// Keep track of each player's dice rolls
+var player1Dice = [];
+var player2Dice = [];
+
+// Keep track of each player's details
+var playerProfiles = [
+  {id: 1, diceNum: 0, score: 0},
```
Comment (kai-rocket) [inline script.js:32]: What does `diceNum` represent? should that be explained?

File: script.js [hunk 2]
```diff
@@ -1,4 +1,307 @@
...
+ * In the event of a tie, Player 2 wins.
+ */
+var determineWinner = function () {
+  // if the first player's diceNum is larger than second player's, return his id
+  if (playerProfiles[0].diceNum > playerProfiles[1].diceNum) {
+    return playerProfiles[0].id;
+  }
+  return playerProfiles[1].id;
+};
+/**
```
Comment (kai-rocket) [inline script.js:155]: Please have consistent spacing between functions

File: script.js [hunk 3]
```diff
@@ -1,4 +1,307 @@
...
+  // We use the ternary operator on the following line for more concise syntax.
+  // https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Conditional_Operator
+  // The example below is equivalent to:
+  // var diceArray;
+  // if (currPlayer === 1) { diceArray = player1Dice; } else { diceArray = player2Dice; }
+  var diceArray = currPlayer === 1 ? player1Dice : player2Dice;
+  var playerNum;
+  // Sort the array such that it can order any number of elements in ascending order
+  sortAnArray(diceArray);
+  //Alternatively, instead of writing our own sorting algorithm to sort arrays, we can use a native native JS method: diceArray.sort()
```
Comment (kai-rocket) [inline script.js:118]: Spacing after `//`. Consistency is key. Prettier should take care of this, please use prettier formatting for Basics.

**Review by rocketkai — COMMENTED** [review]
Thanks @jeremylim91! CC @awongh

**Pull request closed without merging**